tests/test_endpoints.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import os
+from dotenv import load_dotenv
+from itertools import islice
+import pytest
+
+load_dotenv('./env')
+FLASK_PORT = int(os.getenv('FLASK_PORT'))
+
+
+def test_labels(file_path: str):
+    global FLASK_PORT
+
+    if not os.path.exists(file_path):
+        print(f"This file doesn't exist: {file_path}")
+        return dict()
+
+    url = f"http://localhost:{FLASK_PORT}/match"
+    response = requests.post(url, files={'file': open(file_path, 'rb')})
+
+    if response.status_code == 200:
+        outp_json = response.json()
+        print(f"Response json is: {outp_json}")
+    else:
+        outp_json = dict()
+        print(f"Request failed with status code: {response.status_code}; file path: {file_path}")
+
+    assert len(outp_json) > 0, "Empty json"
+    assert len(outp_json) == 29, "Wrong size json"
```

Comment:
Test cases should be easy to understand. Eg. what does 29 here means, does it need a comment there to explain this, or self explanatory constant will describe it?

Reply:
as I'm not going to change csv file for the test I'm expecting the same result about the result json size. But if service here (f'http://www.ebi.ac.uk/spot/zooma/v2/api/services/annotate?propertyValue={label}') will be changed the result will be changed also
how is it better to explain here?